src/prpl.ts

```diff
@@ -91,11 +91,22 @@ export function makeHandler(rootDir?: string, config?: ProjectConfig): (
 }
 
 class Build {
+  public pushManifest?: push.PushManifest;
+
   constructor(
       private configOrder: number,
       public requirements: Set<capabilities.BrowserCapability>,
       public entrypoint: string,
-      public pushManifest?: push.PushManifest) {
+      buildDir: string) {
+    // TODO Push manifest location should be configurable.
+    const pushManifestPath = path.join(buildDir, 'push-manifest.json');
+    if (fs.existsSync(pushManifestPath)) {
+      console.info(`Detected push manifest "${pushManifestPath}".`);
+      // Note this constructor throws if invalid.
```

Comment:
Who catches the exception? Does the server exit? Seems like it'd be best to catch, log, and keep going